### PR TITLE
[FE:HM-3] - Express Server for run App from Dist folder

### PR DESCRIPTION
**Pull request by ewgenij87snwork**

### CODE REVIEW

**Comment by ewgenij87snwork** [comment]
page_

**Comment by ewgenij87snwork** [comment]
Display errors from Server:

error_handler

File: src/api/api.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+router.get('/posts', async (req, res, next) => {
+  try {
+    res.header('Content-Type', 'application/json');
+    res.sendFile(__dirname + '/mock/post-list.mock.json');
+  } catch (e) {
+    next(e);
+  }
+});
+
+router.get('/error', (req, res, next) => {
```
Comment (urban-knight) [inline src/api/api.js:14]: This one intended for test, so better exclude it in future changes

File: server.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const apiRouter = require('./src/api/api');
+
+app.use('/api', apiRouter);
+
+app.get('/', (req, res) => {
+  res.sendFile(__dirname + '/dist/ys-pet/index.html');
+});
+
+app.get('*', (req, res) => {
```
Comment (urban-knight) [inline server.js:16]: This one will send 404 for user accessing angular route first (ie http://mysite.com/feed, considering /feed - angular route for FeedPageComponent). Better send index.html for angular init that will either render component of 404

**Review by afandul — COMMENTED** [review]
Neat solution, pay attention to comments

**Comment by ewgenij87snwork** [comment]
@urban-knight Thank you!

**Pull request merged**